src/background/starterMods.ts

```diff
@@ -79,6 +95,43 @@ function activateModInOptionsState(
   );
 }
 
+function closeSidebarTab(state: SidebarState, id: UUID): SidebarState {
```

Comment:
```suggestion
function closeSidebarTab(state: SidebarState, modComponentId: UUID): SidebarState {
```

Nit: call this param modComponentId to be more descriptive